src/server/report/result.js

```diff
@@ -121,3 +160,35 @@ export function renderResult(resultData, options) {
 	}
 
 }
+
+export function renderBrowserResults(browser, tests, options) {
+	const results = tests.map(t => {
+		const resultData = t.results.find(r => r.name === browser.name);
+		const duration = resultData.duration;
+		const durationClass = {
+			'error': duration >= 1000,
+			'result-duration': true,
+			'pass': duration < 500,
+			'warning': duration >= 500 && duration < 1000
+		};
+		return html`
+			<div class="result-container">
+				<div class="result-test-name">
+					<h3>${t.name}</h3>
+					<div class="${classMap(durationClass)}">${duration}ms</div>
+				</div>
+				${renderResult(resultData, options)}
+			</div>
+		`;
+	});
+	return html`
+		<div class="result-browser padding">
+			${ICON_BROWSERS[browser.name]}
```

Comment:
To save more space, is there a reason the icon and version don't just live in the tab?  I'm guessing because of the case where there's only one browser and are no tabs?

Reply:
Yeah if there's more coming to the area that makes sense, would be interested in Jeff's thoughts/ideas on it eventually